code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -1001,19 +1001,14 @@
 	reagent_state = LIQUID
 	color = "#FFDCFF"
 	taste_description = "stability"
+	var/list/drug_list = list("crank","methamphetamine","space_drugs","psilocybin","ephedrine","epinephrine","stimulants","bath_salts","lsd","thc")
```

Comment:
Not sure about making this a class-level variable. You can declare it inside `on_mob_life` where you need it.
(same with `stimulants_list` of nanocalcium)

Reply:
Echoing what Farie said on Discord.

Static declaration on a proc make it impossibel to change for admin, and also means any subtype cannot inherit or change it easily. It is better to leave it this way